src/customJs/properties/promo_codes/PromoCodesWidget.test.js

```diff
@@ -30,8 +31,10 @@ describe('Promocode widget', () => {
     prepareUnlayerGlobalObject();
     render(<PromoCodesWidget {...unlayerPropertyProps} />);
 
-    const container = await screen.findByRole('container');
-    expect(container).toBeDefined();
+    const noPromotioncodesMsg = await screen.findByText(
+      messages_es['_dp.promo_codes_not_availables'],
+    );
```

Comment:
No se si vale la pena probar el contenido de esta forma.

Una opción sería probar el contenido en español directamente:

```suggestion
    const noPromotioncodesMsg = await screen.findByText(
      'No se encontraron códigos de promoción disponibles en tu tienda',
    );
```

Otra, probar que hay un texto, sin validar exactamente cual.

La otra, si realmente queremos probar por key, es dejarlo así y en el futuro reproducir en este repo el [componente `TestDopplerIntlProvider`](https://github.com/FromDoppler/doppler-editors-webapp/blob/main/src/components/i18n/TestDopplerIntlProvider.tsx) que directamente devuelve la key.

Reply:
Agregue test por contenidos, en el test que mencionas quise asegurar me que exista un texto independiente del idioma y el contenido que informe al usuario que no existen codigos disponibles

Reply:
ok